app/dashboard/template.hbs

```diff
@@ -84,15 +84,20 @@
                                             </div>
                                             <div class="row">
                                                 <div class="col-xs-12 f-w-xl node-sort-dropdown text-right">
-                                                    <span>
-                                                        {{!-- {{sort-button sortAction=(action 'sort') curOrder=sortOrder}} --}}
-                                                    </span>
-                                                    <label>
-                                                        <select onchange={{action "sort" value="target.value"}}>
-                                                            <option selected={{if (eq sortBy 'title') 'selected'}} value="title">{{t 'general.title'}}</option>
-                                                            <option selected={{if (eq sortBy 'last_logged') 'selected'}} value="last_logged">{{t 'general.modified'}}</option>
-                                                        </select>
-                                                    </label>
+                                                    <select aria-label={{t 'general.sort'}} onchange={{action "sort" value="target.value"}}>
+                                                        <option selected={{if (eq sortBy 'title') 'selected'}} value="title">
```

Comment:
Looks like this bug existed before, but this should be `sort`, not `sortBy`. Also, `sort` should have an initial value (probably `-last_logged`, to match the API's default sort). Otherwise, it shows "Title (asc)" at first, when it's actually sorting by modified (desc).